stdlib/configparser.pyi

```diff
@@ -143,9 +143,11 @@ class RawConfigParser(_Parser):
     ) -> _T: ...
     # This is incompatible with MutableMapping so we ignore the type
     @overload  # type: ignore[override]
-    def get(self, section: str, option: str, *, raw: bool = ..., vars: _Section | None = ...) -> str: ...
+    def get(self, section: str, option: str, *, raw: bool = ..., vars: _Section | None = ...) -> str | None: ...
     @overload
-    def get(self, section: str, option: str, *, raw: bool = ..., vars: _Section | None = ..., fallback: _T) -> str | _T: ...
+    def get(
+        self, section: str, option: str, *, raw: bool = ..., vars: _Section | None = ..., fallback: _T
```

Comment:
Looking at zproject/config.py in zulip (from last mypy_primer output), mypy seems to think that `_T` is `str`, and complains when you do `fallback=None`. I will try to fix that.

```suggestion
        self, section: str, option: str, *, raw: bool = ..., vars: _Section | None = ..., fallback: _T | None
```

Reply:
I reverted this. Returning a possibly `None` value is good if you pass `fallback=None`.